deployer/tasks/setup.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Deployer;
+
+use Deployer\Task\Context;
+use Symfony\Component\Yaml\Yaml;
+
+desc('Setup redaxo instance');
+task('setup', new class {
+    public function __invoke()
+    {
+        $this->checkConfigYml();
+    }
+
+    private function checkConfigYml()
+    {
+        cd('{{release_path}}');
+
+        if (test('[ -f {{data_dir}}/core/config.yml ]')) {
+            return;
```

Comment:
Message an den user

Reply:
Ich hab den Befehl ja in den normalen Deployment-Ablauf integriert, daher möchte ich da glaube ich nicht immer ne Message bekommen. Der Normalfall wird ja sein, dass die config.yml vorhanden ist.
Oder findest du das blöd, es in den normalen Ablauf zu integrieren?
Mein Gedanke war halt, dass man einfach ganz normal deployen kann, auch beim ersten Mal, und dann halt die fehlenden Dinge abgefragt werden.

Reply:
Ich würde setup separat lassen, da dort vermutlich noch mehr checks reinkommen die dann ein normales deployment unnötig verlangsamen oder ggf sofar abbrechen/auf fehler laufen können